tests/components/delete_modal_trigger.test.jsx

```diff
@@ -106,4 +106,56 @@ describe('components/DeleteModalTrigger', () => {
         wrapper.find(ConfirmModal).first().props().onKeyDown({keyCode: Constants.KeyCodes.TAB});
         expect(onDelete).not.toHaveBeenCalled();
     });
+
+    test('should match state when handleOpenModal is called', () => {
+        class ChildModal extends DeleteModalTrigger {}
+
+        const wrapper = shallow(
+            <ChildModal onDelete={jest.fn()}/>
+        );
+
+        const preventDefault = jest.fn();
+        wrapper.setState({showDeleteModal: false});
+        wrapper.instance().handleOpenModal({preventDefault});
+        expect(wrapper.state('showDeleteModal')).toEqual(true);
+        expect(preventDefault).toHaveBeenCalledTimes(1);
```

Comment:
Everything looks great. You could if you wanted to skip the check that `preventDefault` is called because it's arguably testing the internal workings of *how* the click works rather than *what* it does (sets state) but it's not wrong or anything and you may have other reasons why it's important to check that I don't know about 👍 . Nice work.

Reply:
I agree that it might not be necessary but while writing the test, I realized that we have all over in our codebase where we passed an event to another function and sometimes cascaded several times before the last function calls the `preventDefault`. That's why I thought, I should add this assertion whenever possible.